app/controllers/idv/hybrid_mobile/socure/document_capture_controller.rb

```diff
@@ -4,11 +4,10 @@ module Idv
   module HybridMobile
     module Socure
       class DocumentCaptureController < ApplicationController
-        include Idv::AvailabilityConcern
```

Comment:
Concerns in the `Idv::` scope don't need the full, explicit name so I'm matching the other concerns (e.g., RenderConditionConcern)